docs/integrations/integration_template.md

```diff
@@ -0,0 +1,111 @@
+---
+title: (Template) Integrating { PRODUCT NAME } With Great Expectations
+authors:
+    name: AUTHOR NAME
+    url: https://superurl.com
+---
+
+:::info
+* Maintained By: Company Name
+* Status: Alpha | Beta | Production/Mature
+* Testing: https://URL-to-CI-If-Public.io (otherwise remove)
+* Support/Contact: https://yoururl.com
+:::
+
+### Introduction
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Viverra nam libero justo laoreet. Hendrerit gravida rutrum quisque non. Mauris pellentesque pulvinar pellentesque
+habitant morbi. Sed enim ut sem viverra aliquet eget. A diam maecenas sed enim ut sem viverra aliquet eget.
+Non sodales neque sodales ut etiam sit amet nisl. Fermentum iaculis eu non diam.
+
```

Comment:
The draft I'd worked on had a section on "Dev Loops unlocked by the integration", basically the hero story enabled by the using the tool. We may want to call that out explicitly, or we can work it into a section, but I like the idea of a bold diagram explaining how the tools work together.